src/coprocessor/dag/executor/scan.rs

```diff
@@ -33,7 +33,7 @@ pub trait InnerExecutor {
     fn is_point(&self, range: &KeyRange) -> bool;
 
     // indicate which scan it will perform, Table or Index, this will pass to Scanner.
-    fn scan_on(&self) -> ScanOn;
+    fn scan_on(&self) -> super::super::scanner::ScanOn;
```

Comment:
Could we import `super::super::scanner::ScanOn` it's too long..

Reply:
Good idea. Let's change it later because I have already proposed long chaining PRs based on this.. 😭Change it here will cause a lot of conflicts to solve.